iota/api.py

```diff
@@ -252,18 +323,17 @@ def check_consistency(self, tails):
         one) to be confirmed, or not (because it conflicts with another
         already confirmed transaction).
 
-        :param tails:
+        :param Iterable[TransactionHash] tails:
             Transaction hashes. Must be tail transactions.
 
         :return:
-            Dict with the following structure::
+            ``dict`` with the following structure::
 
                 {
                     'state': bool,
-
-                    'info': str,
-                        This field will only exist set if ``state`` is
-                        ``False``.
+                        Wether tails resolve to consistent ledger.
+                    'info': Text,
+                        This field will only exist if 'state' is False.
```

Comment:
We can't control the response from the node in ``check_consistency``, so to make it consistent, I suggest we implement the same behavior in ``is_promotable``, that is to do
```
if response['promotable']:
    del response['info']
```
instead of the current ``response['info']=None`` in https://github.com/iotaledger/iota.py/blob/a68254d040dcf9a581a851f5953e056ff538897f/iota/commands/extended/is_promotable.py#L96